.github/pull_request_template.md

```diff
@@ -0,0 +1,96 @@
+    Please answer all questions in this section for all PRs.
+--->
+
+**PR Type**
+<!---
+    Please check the corresponding box.
+
+    A "patch" is a PR which touches relatively few lines of code. Patches
+    usually address bugs, minor performance issues, typos, clarify
+    documentation, etc. Most patches are ready to go in a day or two.
+
+    A "feature" is a PR which adds a major new capability, massively overhauls
+    an existing feature, writes entirely new documentation pages, or optimizes
+    an extensive algorithm. Features usually take at least a week to implement.
+
+    "Breaking change" is a PR which will break existing user-facing code. These
+    types of PRs must be discussed in advance. They may be very small, or very
+    extensive PRs depending on the change.
+--->
+
+- [ ] Breaking change
+- [ ] Feature
+- [ ] Patch
+
+**Brief Description**
+<!---
+    In a couple sentences, describe what this pull request will accomplish. If
+    there is a corresponding issue please link to it with
+    [closing words](
+        https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-
+        using-a-keyword)
+    as appropriate. If the goal is more complicated than can be articulated in
+    a few sentences, please first open an issue and explain it in detail
+    there.
+--->
+
+**Not In Scope**
+<!---
+    Some features have obvious extensions or use cases. If you're only targeting
+    a specific use case and don't want to worry about other use cases in this
+    PR please make that clear.
+--->
+
+
+
+**PR Checklist**
+
+<!---
+    This checklist is meant for developers who are part of the NWChemEx-Project
+    organization. If you are an outside collaborator, who only occasionally
+    contributes little tweaks, we're just happy to get your contribution. The
+    reviewers will happily make any changes needed to bring your contribution
+    up to snuff.
+
+    For developers who are part of the organization, and outside collaborators making frequent or large contributions, know that the NWChemEx-Project
+    organization strives to be an exemplar of code quality. Unless you have
+    advance approval, your PR will not be merged until all relevant items on the
+    checklist have been addressed. That said, if you're new, we're certainly
+    willing help out, answer questions, and point you to the right resources.
+--->
+
+- [ ] Documentation (comments, API, user, design, and developer as applicable)
+- [ ] Testing (unit, integration, performance, as applicable)
+- [ ] Code adheres to organization standards
+
+# Draft PRs Only
+<!---
+    We strongly encourage all PR authors to open PRs as early as possible.
+    When you do that your PR is usually not ready for review. Fill this section
+    out in that case. Note we encourage breaking PRs down into chunks which
+    can be merged within two weeks. If you expect that your TODO list is too
```

Comment:
Review time is part of it, but I am really just worried about the PRs sitting around for months. When PRs sit around for a long time:

- PR feature is typically overlooked in ongoing design
- Easy for new changes to master to break correctness/performance of PR
- Prohibits others from leveraging the feature (even if it's just having a hook to the feature)
- Causes PR author to spend large amounts of time just keeping up
- Reviewers often forget subtleties regarding the PR like why something was/wasn't done
- The PR tends to contain a large amount of code, making it harder to review
- It tends to increase the amount of technical debt by justifying "let's just merge this"

Admittedly this is less of an issue right now since there are only a few people poking around in the code, but if master is constantly changing these issues will become more pronounced.

FTR, two-weeks is a starting point; I'm not wed to it.

Reply:
I added an FAQ section and a question pertaining to issue vs. PR. I also made sure these bullet points made it into PR policy.

Reply:
I am good with this, based on our off-line discussion and the changes that you made in the other documentation.